user/send-email.sh

```diff
@@ -28,7 +28,7 @@ ENC_PASSWORD="$5"   # PGP encrypted password
 # PASSWORD=$(base64 --decode | keybase pgp decrypt)
 
 # Create a temporary file to store the encrypted binary data.
-password_file=$(mktemp "password.${USERNAME}.gpg")
+password_file=$(mktemp "password.${USERNAME}.gpg.XXXXX")
```

Comment:
why the XXXX suffix?

Reply:
mktemp returns error on ubuntu:
https://github.com/rtomayko/shocco/issues/5

Reply:
Can you include that link as the reason in a comment above line 31?